features/post_filter_submit.feature

```diff
@@ -0,0 +1,131 @@
+Feature: Post Filter Private Endpoints Not Enabled
+
+  Background:
+    Given private endpoints are not enabled
+
+    And I have these filters:
```

Comment:
I think it may still be good to keep them there and then change the tests once the mock code is removed.
The tests should fail and we would have to fix them again.

Reply:
the tests will still fail when the mock will be replaced by actual irrespective of the filters being in db or not